ostree-simplified-installer.sh

```diff
@@ -1385,6 +1394,11 @@ sudo composer-cli compose delete "${COMPOSE_ID}" > /dev/null
 sudo composer-cli blueprints delete upgrade > /dev/null
 
 greenprint "🗳 Upgrade ostree image/commit"
+# Update default Fedora's repository
+if [[ "$ID" == fedora ]]; then
+    sudo ssh "${SSH_OPTIONS[@]}" -i "${SSH_KEY}" ${BLUEPRINT_USER}@${PUB_KEY_GUEST_ADDRESS} "echo ${EDGE_USER_PASSWORD} |sudo -S ostree remote delete ${REF_PREFIX}"
```

Comment:
It's not `PUB_KEY_GUEST_ADDRESS` but `ROOT_CERT_GUEST_ADDRESS`.

Reply:
Sure, done. Thanks!